package.json

```diff
@@ -69,6 +69,14 @@
       "es2015"
     ]
   },
+  "nyc": {
+    "extension": [
+      ".raw"
```

Comment:
Hmm, nope ... it breaks. `nyc` seems to expect some kind of extra for that extension then ...